src/containers/List/ListWrapper.tsx

```diff
@@ -85,24 +86,12 @@ const ListWrapper: React.FC = () => {
     }
   }
 
-  const handleExpansion = (serialNumber: string) => {
+  const handleExpansion = (application: ApplicationListRow) => {
     if (!applicationsRows) return
```

Comment:
You could *probably* lose this line too, since there's no way `applicationRows` couldn't exist if you've just passed an `application` (which is inside `applicationRows`) to this function.

Reply:
Yep! I see. Thanks

Reply:
Ah, nah. Then it would required setting to ApplicationListRow[] on the spread which would be less visible. So keeping as it is